code/src/java/pcgen/core/analysis/SubstitutionLevelSupport.java

```diff
@@ -45,14 +43,7 @@ private static boolean levelArrayQualifies(int level, final PlayerCharacter pc,
 		final PCClassLoader classLoader = new PCClassLoader();
 		PCClass dummyClass = new PCClass();
 
-		try
-		{
-			classLoader.parseLine(Globals.getContext(), dummyClass, aLine, tempSource);
-		}
-		catch (PersistenceLayerException e)
-		{
-			Logging.errorPrint("Unable to parse line from levelArray: " + aLine);
-		}
+		classLoader.parseLine(Globals.getContext(), dummyClass, aLine, tempSource);
```

Comment:
I'm concerned this changes the function of this method - previously, this exception would cause an error that is local to this method; now the exception will bubble up to a higher level; potentially causing it to process many less lines in a file.  The design of this (and many other items where you are removing reactions to PersistenceLayerException) are to try to have a minimal failure - as little as possible should be not processed.  In this new case, if a failure is encountered, does it completely stop processing the file (or the entire load?)

Reply:
We used to call this 'heroic mode' ;-), the loaders keep on chugging even on glitchy data.

Reply:
it doesn't change any functionality.  the exception is checked and you can investigate to see that it cannot be thrown in those places. this has been dead code for a while.  can you show that the exception is actually thrown?